src/power_grid_model_io/config/excel/vision_en.yaml

```diff
@@ -244,8 +255,10 @@ grid:
       r_grounding_to: 0
       x_grounding_to: 0
       extra:
-        - Name
+        - Node.Number
```

Comment:
I guess the same here (Node.Number / Subnumber)

Reply:
Whet do you mean? Here it does say Node.Number and Subnumber, right?